### PR TITLE
Update CDITestCheckpoint to remove incorrect server configuration

### PR DESCRIPTION
**Pull request by SmithaSubbarao**
- [ ] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [ ] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [ ] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

**Comment by SmithaSubbarao** [comment]
Test info:

1. CDITestCheckpoint:
```
 [startAnt]     [mkdir] Created dir: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html
 [startAnt] [junitreport] Processing /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit/TESTS-TestSuites.xml to /tmp/null1783655175
 [startAnt] [junitreport] Loading stylesheet jar:file:/usr/share/java/ant-junit-1.10.12.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
 [startAnt] [junitreport] Transform time: 887ms
 [startAnt] [junitreport] Deleting: /tmp/null1783655175
 [startAnt]      [echo] HTML report is available at /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html/index.html
 [startAnt] 
 [startAnt] BUILD SUCCESSFUL
 [startAnt] Total time: 1 second

archiveReports:

execute:

BUILD SUCCESSFUL
Total time: 1 minute 16 seconds

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.10.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 5m 9s
358 actionable tasks: 69 executed, 289 up-to-date
```

2. JdbcConfig:
```
    [mkdir] Created dir: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/build/tmp
     [echo] framework.maxheap=-Xmx1024m
    [junit] Running batch.fat.junit.FATSuite
    [junit] Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 293.43 sec
     [echo] Finished running batch.fat.junit.FATSuite
     [echo] XML report is available at /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit

generateReports:
 [startAnt] Buildfile: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/src/ant/launch.xml
 [startAnt] 
 [startAnt] reallyGenerateReports:
 [startAnt]      [echo] Generating an HTML report based on the existing XML report
 [startAnt]     [mkdir] Created dir: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html
 [startAnt] [junitreport] Processing /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit/TESTS-TestSuites.xml to /tmp/null1810587038
 [startAnt] [junitreport] Loading stylesheet jar:file:/usr/share/java/ant-junit-1.10.12.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
 [startAnt] [junitreport] Transform time: 1080ms
 [startAnt] [junitreport] Deleting: /tmp/null1810587038
 [startAnt]      [echo] HTML report is available at /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html/index.html
 [startAnt] 
 [startAnt] BUILD SUCCESSFUL
 [startAnt] Total time: 2 seconds

archiveReports:

execute:

BUILD SUCCESSFUL
Total time: 5 minutes 1 second

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.10.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 6m 14s
358 actionable tasks: 66 executed, 292 up-to-date
```

3. Full CDITestCheckpoint: ```./gradlew com.ibm.ws.jbatch.open_fat:buildandrun -Dfat.test.class.name=CDITestCheckpoint -Dfat.test.mode=FULL```
```
    [mkdir] Created dir: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/build/tmp
     [echo] framework.maxheap=-Xmx1024m
    [junit] Running batch.fat.junit.FATSuite
    [junit] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 187.433 sec
     [echo] Finished running batch.fat.junit.FATSuite
     [echo] XML report is available at /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit

generateReports:
 [startAnt] Buildfile: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/src/ant/launch.xml
 [startAnt] 
 [startAnt] reallyGenerateReports:
 [startAnt]      [echo] Generating an HTML report based on the existing XML report
 [startAnt]     [mkdir] Created dir: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html
 [startAnt] [junitreport] Processing /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit/TESTS-TestSuites.xml to /tmp/null84373458
 [startAnt] [junitreport] Loading stylesheet jar:file:/usr/share/java/ant-junit-1.10.12.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
 [startAnt] [junitreport] Transform time: 968ms
 [startAnt] [junitreport] Deleting: /tmp/null84373458
 [startAnt]      [echo] HTML report is available at /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html/index.html
 [startAnt] 
 [startAnt] BUILD SUCCESSFUL
 [startAnt] Total time: 2 seconds

archiveReports:

execute:

BUILD SUCCESSFUL
Total time: 3 minutes 15 seconds

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.10.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 7m 6s
358 actionable tasks: 82 executed, 276 up-to-date
```

4. Full batch bucket with Checkpoint only tests, ```./gradlew com.ibm.ws.jbatch.open_fat:buildandrun -Dfat.test.mode=FULL -Dfat.test.run.checkpoint.only=true```

```
    [mkdir] Created dir: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/build/tmp
     [echo] framework.maxheap=-Xmx1024m
    [junit] Running batch.fat.junit.FATSuite
    [junit] Tests run: 27, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1,020.01 sec
     [echo] Finished running batch.fat.junit.FATSuite
     [echo] XML report is available at /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit

generateReports:
 [startAnt] Buildfile: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/src/ant/launch.xml
 [startAnt] 
 [startAnt] reallyGenerateReports:
 [startAnt]      [echo] Generating an HTML report based on the existing XML report
 [startAnt]     [mkdir] Created dir: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html
 [startAnt] [junitreport] Processing /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit/TESTS-TestSuites.xml to /tmp/null1864785575
 [startAnt] [junitreport] Loading stylesheet jar:file:/usr/share/java/ant-junit-1.10.12.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
 [startAnt] [junitreport] Transform time: 1010ms
 [startAnt] [junitreport] Deleting: /tmp/null1864785575
 [startAnt]      [echo] HTML report is available at /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html/index.html
 [startAnt] 
 [startAnt] BUILD SUCCESSFUL
 [startAnt] Total time: 2 seconds

archiveReports:

execute:

BUILD SUCCESSFUL
Total time: 17 minutes 7 seconds

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.10.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 17m 45s
358 actionable tasks: 66 executed, 292 up-to-date
```

6. Full JdbcConfigTestCheckpoint:

```
    [mkdir] Created dir: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/build/tmp
     [echo] framework.maxheap=-Xmx1024m
    [junit] Running batch.fat.junit.FATSuite
    [junit] Tests run: 24, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 846.41 sec
     [echo] Finished running batch.fat.junit.FATSuite
     [echo] XML report is available at /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit

generateReports:
 [startAnt] Buildfile: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/src/ant/launch.xml
 [startAnt] 
 [startAnt] reallyGenerateReports:
 [startAnt]      [echo] Generating an HTML report based on the existing XML report
 [startAnt]     [mkdir] Created dir: /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html
 [startAnt] [junitreport] Processing /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit/TESTS-TestSuites.xml to /tmp/null305164869
 [startAnt] [junitreport] Loading stylesheet jar:file:/usr/share/java/ant-junit-1.10.12.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
 [startAnt] [junitreport] Transform time: 927ms
 [startAnt] [junitreport] Deleting: /tmp/null305164869
 [startAnt]      [echo] HTML report is available at /root/open-liberty-instanton-setup/ub2204/open-liberty/dev/com.ibm.ws.jbatch.open_fat/build/libs/autoFVT/results/junit.html/index.html
 [startAnt] 
 [startAnt] BUILD SUCCESSFUL
 [startAnt] Total time: 2 seconds

archiveReports:

execute:

BUILD SUCCESSFUL
Total time: 14 minutes 13 seconds

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.10.1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 14m 49s
358 actionable tasks: 66 executed, 292 up-to-date
```

**Comment by SmithaSubbarao** [comment]
#build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=8d234d7f-d556-492e-ac8e-208c5e0cb1aa) - :x: completed with errors/failures.)

_Note: Target locations of links might be accessible only to IBM employees._

File: dev/com.ibm.ws.jbatch.open_fat/fat/src/batch/fat/junit/CDITestCheckpoint.java [hunk 1]
```diff
@@ -41,7 +41,6 @@ public class CDITestCheckpoint extends BatchFATHelper {
     @BeforeClass
     public static void setup() throws Exception {
 
-        server = LibertyServerFactory.getLibertyServer("checkpointbatchFAT");
         // Just happens to be a config that works we could reuse.  Could rename.
         BatchFATHelper.setConfig("CDITestCheckpoint/server.xml", testClass);
```
Comment (tjwatson) [inline dev/com.ibm.ws.jbatch.open_fat/fat/src/batch/fat/junit/CDITestCheckpoint.java]: Originally I had suggested that we continue to use the `checkpointbatchFAT` server but move the file `dev/com.ibm.ws.jbatch.open_fat/publish/files/CDITestCheckpoint/server.xml` to `dev/com.ibm.ws.jbatch.open_fat/publish/servers/checkpointbatchFAT/server.xml` and remove this `setConfig` call.

I think that would be more clean than copying the config over into the `batchFAT` server assigned by the super class.

**Review by tjwatson — CHANGES_REQUESTED** [review]

**Comment by SmithaSubbarao** [comment]
#build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=f4ba31a4-8bac-4b40-88af-e3b342cf3d5a) - :white_check_mark: completed successfully!)

_Note: Target locations of links might be accessible only to IBM employees._

File: dev/com.ibm.ws.jbatch.open_fat/publish/servers/checkpointbatchFAT/server.xml [hunk 2]
```diff
@@ -1,51 +1,29 @@
...
-    <application type="war" id="snoop" name="snoop" location="${shared.app.dir}/webcontainer/snoop" />
-    -->
-    <include location="../fatTestPorts.xml"/>
-    
-    <!-- NOW THIS MESSAGE CHANGED -->
-    
-     <!-- batch container persistence config -->
-     <!--
+
+    <include location="../fatTestPorts.xml" />
```
Comment (tjwatson) [inline dev/com.ibm.ws.jbatch.open_fat/publish/servers/checkpointbatchFAT/server.xml:3]: You should delete the original file at `dev/com.ibm.ws.jbatch.open_fat/publish/files/CDITestCheckpoint/server.xml`

**Review by tjwatson — APPROVED** [review]

**Comment by SmithaSubbarao** [comment]
#build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=43eb2748-0bc8-4008-b359-2896b55efcff) - :white_check_mark: completed successfully!)

_Note: Target locations of links might be accessible only to IBM employees._

**Pull request closed without merging**